.github/ISSUE_TEMPLATE/enhancement_or_feature_request.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Enhancement or feature request.
+description: Suggest an enhancement or feature for Sublime Text core.
+body:
+- type: textarea
+  attributes:
+    label: Problem description
+    description: |
+      A description of a problem, workflow or integration that your suggestion would solve. If the problem is OS specific, include that information here.
+
+      1. Make sure that you first check if your feature/enhancement request hasn't already been reported. It saves a bit of work on our end.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Preferred solution
+    description: "A description of what changes should be made to Sublime Text to solve the problem."
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Alternatives
+    description: "A description of any alternative solutions or enhancements considered."
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Additional Information (optional)

```

Comment:
Sorry about my absence. I removed it because
1. it's not in the bug report template
2. required fields are already marked with a red asterisk

![2021-10-31_00-10-20](https://user-images.githubusercontent.com/931051/139559712-28fd5d5f-7d6d-450d-b51c-46ed98be46f3.png)

Reply:
Makes sense.